CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ set(${LIBRARY_TARGET_NAME}_SRC
   src/Solver.cpp)
 
 set(${LIBRARY_TARGET_NAME}_HDR
-  include/OsqpEigen.h
+  include/OsqpEigen/OsqpEigen.h
```

Comment:
Isn't this breaking the compatibility with other projects?

Reply:
Probably not, as we were already doing ``#include <OsqpEigen/OsqpEigen.h>`` as in https://github.com/robotology/walking-controllers/blob/master/modules/Walking_module/include/WalkingQPInverseKinematics_osqp.hpp

Reply:
The installation layout was already `<install_prefix>/OsqpEigen/OsqpEigen.h`, this just aligns the source layout with the  install layout.